src/util/sandbox/fd_sandbox.c

```diff
@@ -327,3 +327,26 @@ fd_sandbox_getpid( void ) {
 
   return result;
 }
+
+ulong
+fd_sandbox_gettid( void ) {
+  char tid[ 27 ] = {0}; /* 10 characters for INT_MAX, twice, + /task/ and then a NUL terminator. */
+  long count = readlink( "/proc/thread-self", tid, sizeof(tid) );
```

Comment:
`readlink` does not append a nul terminator, so you should probably pass in `sizeof(tid)-1` or set `tid[26]='\0'`.

Reply:
Although I think your computation of 27 is right, so it should always leave a char at the end that is 0.